### ISSUE TITLE
Друг, ты живой? что-то обнов нету давно утилита пишет чекнуть обнову

### ISSUE THREAD

**Issue by Glazami** [ValdikSS/blockcheck#90]
Кэп?!

**Comment by Glazami** [comment]
Ваш провайдер полностью блокирует доступ к https.
isup.me даёт неожиданные ответы или недоступен. 
пс пробовал зайти сайт работает вроде 
программу говорит что провайдер блокирует по ip рекомендуется любой способ.
Какой способ например? doh/dot не помогает. vpn не вариант.
![1](https://user-images.githubusercontent.com/65298911/120342526-ec818d00-c308-11eb-83b8-e523a9b54f91.PNG)
![2](https://user-images.githubusercontent.com/65298911/120342534-ee4b5080-c308-11eb-9d21-edc15d576289.PNG)

**Comment by Glazami** [comment]
Можно как-то на уровне роутера настроить из под openwrt или padavan?

**Comment by sxiii** [comment]
> Можно как-то на уровне роутера настроить из под openwrt или padavan?

Да, в OpenWRT всё можно настроить что угодно, это практически полноценный линукс, пакетов чуть чуть меньше, но никто не запрещает собирать из исходников при необходимости

**Comment by Glazami** [comment]
> 
> 
> > Можно как-то на уровне роутера настроить из под openwrt или padavan?
> 
> Да, в OpenWRT всё можно настроить что угодно, это практически полноценный линукс, пакетов чуть чуть меньше, но никто не запрещает собирать из исходников при необходимости

А куда копать? Где-то есть гайды?

**Comment by sxiii** [comment]
Гайдов множество в сети, гугли "OpenWRT" + что ты хочешь настроить... + "set up manual"... @Glazami

**Comment by IRainman** [comment]
С автором всё хорошо, просто проект потерял актуальность. По-хорошему он должен стать архивом.
